frontend/src/components/dashboard/MuseumGallery.js

```diff
@@ -233,30 +242,36 @@ export default function MuseumGallery() {
         >
           <MenuItem
             onClick={(event) => handleChangeSortBy(event, 'relevance')}
+            aria-label="Sort by: Relevance"
```

Comment:
Have you tested these with a screen reader? I was under the impression that the menu item text would be read by the screen reader too so the aria label would be partially repetitive. aria-labelledby might be a choice. If you've already tested this then disregard this.